LICENSE

```diff
@@ -0,0 +1,24 @@
+Open Insights
+Version: <%= pkg.version %>
+Generated: <%= moment().format("YYYY-MM-DD") %>
+https://github.com/openinsights/openinsights
+
+Copyright (c) <%= moment().format("YYYY") %>, CBS Interactive Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
This looks like an MIT license, which sounds good to me. @vnsparacio does an MIT license work for the company?

Reply:
I will check in with legal on this